src/main/resources/main.css

```diff
@@ -36,7 +36,7 @@
    -fx-background-color: transparent;
 }
 
-#otherActionlog #textlabel {
+#otherActionlog #chatField {
```

Comment:
try to be more generic. Ids should be unique for css (=>only one element should match #otherActionlog). If you use classes then styles like -fx-background-radius could be applied to like `.messagebox` instead of being doubledefined for `#otherActionlog` and `#myActionlog`